src/common/ConfUtils.cc

```diff
@@ -252,10 +252,14 @@ trim_whitespace(std::string &str, bool strip_internal)
 std::string ConfFile::
 normalize_key_name(const std::string &key)
 {
-  string k(key);
-  ConfFile::trim_whitespace(k, true);
-  std::replace(k.begin(), k.end(), ' ', '_');
-  return k;
+  if (key.find_first_of(" \t\r\n\f\v\xa0") != string::npos) {
```

Comment:
nit, could early return. less code churn, less nesting levels.
```c++
{
  if (key.find_first_of(" \t\r\n\f\v\xa0") == key.npos) {
    return key;
  }
  string k(key);
  // ...
  return k;
}

Reply:
You're right, changed it.